lib/model.js

```diff
@@ -676,18 +676,14 @@ class Model {
    * @return {Model}
    */
   static init(attributes, options) { // testhint options:none
-    options = options || {};
-
-    if (!options.sequelize) {
+    if (!options || !options.sequelize) {
       throw new Error('No Sequelize instance passed');
     }
     this.sequelize = options.sequelize;
 
     const globalOptions = this.sequelize.options;
 
-    if (globalOptions.define) {
-      options = Utils.merge(globalOptions.define, options);
-    }
+    options = Utils.merge(_.cloneDeep(globalOptions.define), options);
```

Comment:
What happens now if `globalOptions.define` is `undefined`? I think previous version was better